### PR TITLE
Implementation of Folder Jumping

### PR DESCRIPTION
**Pull request by sriio**
Simple implementation of [ Folder Jumping](https://github.com/blog/1877-folder-jumping)

Comes with this [PR](https://github.com/go-gitea/git/pull/23)

### CODE REVIEW

**Comment by andreynering** [comment]
LGTM

Very good. After https://github.com/go-gitea/git/pull/23 being merged, please do update vendor e push another commit:

```
govendor update code.gitea.io/git
```

**Comment by Bwko** [comment]
Great feature, but I personally prefer the Github approach (greying out empty folders). Maybe you could add that?

File: public/css/index.css [hunk 1]
```diff
@@ -1272,6 +1272,9 @@ footer .ui.language .menu {
 .repository.file.list #repo-files-table tr:hover {
   background-color: #ffffEE;
 }
+.repository.file.list #repo-files-table .jumpable-path {
```
Comment (Bwko) [inline public/css/index.css:1275]: Css files get generated by less

File: templates/repo/view_list.tmpl [hunk 2]
```diff
@@ -45,8 +45,15 @@
...
+							{{$subJumpablePath := SubJumpablePath $subJumpablePathName}}
 							<span class="octicon octicon-file-directory"></span>
-							<a href="{{EscapePound $.TreeLink}}/{{EscapePound $subJumpablePathName}}">{{$subJumpablePathName}}</a>
+							<a href="{{EscapePound $.TreeLink}}/{{EscapePound $subJumpablePathName}}">
+								{{if eq (len $subJumpablePath) 2}}
+									<span class="jumpable-path">{{index  $subJumpablePath 0}}</span>{{index  $subJumpablePath 1}}
+								{{else}}
+									{{index $subJumpablePath 0}}
+								{{end}}
+                            </a>
```
Comment (andreynering) [inline templates/repo/view_list.tmpl]: Wrong indentation?

File: public/css/index.css (hunk 1 above)
Comment (sriio) [reply public/css/index.css:1275]: Rule added to the less file

**Comment by bkcsoft** [comment]
@sriio You feel done with this? in that case `govendor update code.gitea.io/git`, squash and force-push :)

File: public/css/index.css (hunk 1 above)
Comment (Bwko) [reply public/css/index.css:1275]: Great :+1:

**Comment by Bwko** [comment]
LGTM

**Pull request merged**